google-cloud-error_reporting/lib/google/cloud/error_reporting/rails.rb

```diff
@@ -64,9 +64,7 @@ class Railtie < ::Rails::Railtie
             service_version = er_config.service_version
 
             error_reporting = V1beta1::ReportErrorsServiceClient.new(
-              channel: channel,
-              app_name: service_name,
-              app_version: service_version
+              channel: channel
```

Comment:
Why app_name and app_version here are removed, should they be placed as lib_name and lib_version?

@hxiong388